.travis.yml

```diff
@@ -24,7 +24,8 @@ env:
     - TASK=mpi METHOD=pip PYTHON_VERSION=3.7
     - TASK=gpu METHOD=source PYTHON_VERSION=3.5
     - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
-    - TASK=r-package
+    - TASK=r-package R_VERSION=3.6
+    - TASK=r-package R_VERSION=4.0
```

Comment:
It's too much I think! Let's keep all possible combinations for #2353. We can test different R versions at Travis and Azure, e.g. 4 here and 3 at Azure or vise versa.

Reply:
ok that's a good point. We're getting the same information from both places. I do like that when Azure is failing and Travis isn't (or vice versa), it gives us some information that an issue might be CI-provider-specific, but that's probably not worth the cost of the duplicated builds.

I'll test R4.0 on Travis and 3.6 on Azure (since Azure Linux builds involve updating that CI container). So I think it would actually be 2 in Travis (Mac + Linux, R4.0) and 2 on Azure (Mac + Linux, R3.6)

Reply:
removed the R 3.6 builds on Travis: https://github.com/microsoft/LightGBM/pull/3065/commits/983dd2d2b573b0909e4bd7c1ba8330eba59685eb

Reply:
I just rebased to master and now all R Mac and Travis jobs run on GitHub Actions. Right now I have the matrix set up to run 8 total tasks, all combinations of:

1. operating system: [Mac, Linux]
2. compiler: [gcc, clang]
3. R versions: [3.6, 4.0]

GitHub Actions gives you 20 concurrent builds so I think having 8 R builds is reasonable given the current pace of development on this project. If it becomes too much, it can definitely be cut down.